src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -9,35 +9,35 @@ namespace Microsoft.DotNet.Tools.Test
     // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs !!!
     internal class FeatureFlag
     {
-        private const string Prefix = "VSTEST_FEATURE_";
+        private static readonly Dictionary<string, bool> FeatureFlags = new();
 
-        public Dictionary<string, bool> FeatureFlags { get; } = new();
+        private const string VSTEST_ = nameof(VSTEST_);
 
-        public static FeatureFlag Default { get; } = new FeatureFlag();
+        public static FeatureFlag Instance { get; } = new FeatureFlag();
 
-        public FeatureFlag()
+        static FeatureFlag()
         {
-            FeatureFlags.Add(ARTIFACTS_POSTPROCESSING, true);
+            FeatureFlags.Add(DISABLE_ARTIFACTS_POSTPROCESSING, false);
```

Comment:
Do not initialize here, we don't know what the user preference is and we have to look it up from the environment when the flag is used in code.

Reply:
I think we can remove the dictionary, now that we have default always `false` semantics we can only check the env var if present and the value, if it's not set is false.